modules/compliance-tailored-profiles.adoc

```diff
@@ -46,7 +46,32 @@ spec:
     rationale: Organizational requirements
     value: permissive
 ----
++
+
+
++
+.Attributes for spec variables
+[cols="1,2a",options="header"]
+|===
+|Attribute
+|Description
+
+|`extends`
+|Name of the `Profile` object upon which this `TailoredProfile` is built.
 
+|`title`
+|Human-readable title of the `TailoredProfile`.
+
+|`disableRules`
+|A list of name and rationale pairs. Each name refers to a name of a rule object that is to be disabled. The rationale value is human-readable text describing why the rule is disabled.
+
+|`enableRules`
+|A list of name and rationale pairs. Each name refers to a name of a rule object that is to be enabled. The rationale value is human-readable text describing why the rule is enabled.
+
+|`description`
+|Human-readable text describing the `TailoredProfile`.
+|===
```

Comment:
Can we have `setValues`attributes description in above table ?

Reply:
Thanks! I've added setValues.